file-formats/desd/type/zif_aff_desd_v1.intf.abap

```diff
@@ -0,0 +1,35 @@
+INTERFACE zif_aff_desd_v1
+  PUBLIC.
+
+  TYPES:
+
+    "! <p class="shorttext">General Information</p>
+    "! General information
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">Default Remote Schema</p>
+      "! Default remote database schema name when configuring the connection
+      "! details of logical external schema.
+      default_remote_schema_name TYPE c LENGTH 256,
```

Comment:
I saw that in the previous AFF review the field is not supposed to be required. Could you please clarify what happens when this field is empty? I still don't understand the workflow of this object.

Reply:
I will mark this comment as Resolved as we are clarified.